cclib/parser/data.py

```diff
@@ -66,6 +66,7 @@ class ccData:
         natom -- number of atoms (integer)
         nbasis -- number of basis functions (integer)
         nmo -- number of molecular orbitals (integer)
+        nmrtensors -- Nuclear magnetic resonance tensors (dict of dicts of array[2])
```

Comment:
`Nuclear magnetic resonance shielding tensors (dict of dicts of array[2])` sounds a little better but this doesn't really matter. 

This dict of dicts is somewhat confusing. Perhaps adding a section in parsed data notes in the docs?